core/scss/utilities/mixins/modular-scale/_modular-margin-bottom.scss

```diff
@@ -0,0 +1,18 @@
+@charset "UTF-8";
+
+//
+// @modular-margin-bottom
+//
+// A mixin for adding modular margin-bottom to an element.
+//
+// $step - Integers from -4 to 5 including 0. Default for $step is 0 which returns the modular spacing base value.
+//
+// Style guide: Mixins.Modular.modular-margin-bottom
+//
+@mixin modular-margin-bottom($step: 0, $grid: $grid-media) {
```

Comment:
I get why you have created this function but it seems overly specific and in the previous rendition of the modular functions I wrote something similar but I was not overly fond of it. 

Do you think we could refactor these modular-margin-vertical modular-margin-bottom to perhaps pass in the css property we would like to have modular value on?

eg: 
```
function modular-value-property($step, $property, $unit) {
... code ...
}
```

usage:
```
p {
  @include modular-value-property(1, 'margin-bottom', 'px')
}
```

Reply:
Yeah, I was thinking all the mixins were a bit redundant too. Sure, I'll rewrite this part to consolidate. Should we support 'px' unit or just allow 'em'?

Reply:
Good question. Let's start with just `em` and do not allow for the unit value in the mixin. We can adjust that later if we need to expand.